Source/WebKit/Shared/Extensions/WebExtensionUtilities.h

```diff
@@ -113,6 +113,14 @@ id toWebAPI(const std::optional<T>& result, UseNullValue useNull = UseNullValue:
     return toWebAPI(result.value());
 }
 
+inline NSDictionary *toWebAPI(const JSON::Object& object)
+{
+    auto jsonString = object.toJSONString();
+    RetainPtr jsonData = [jsonString.createNSString() dataUsingEncoding:NSUTF8StringEncoding];
+    NSDictionary *dictionary = [NSJSONSerialization JSONObjectWithData:jsonData.get() options:NSJSONReadingAllowFragments error:nil];
```

Comment:
`NSJSONReadingAllowFragments` allows results other than `NSDictionary`. Just pass `0` for dictionary.
```suggestion
    NSDictionary *dictionary = [NSJSONSerialization JSONObjectWithData:jsonData.get() options:0 error:nil];
```